packages/renderer/src/lib/preferences/item-formats/NumberItem.spec.ts

```diff
@@ -0,0 +1,29 @@
+import '@testing-library/jest-dom/vitest';
+import { test, expect, vi, beforeAll } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import type { IConfigurationPropertyRecordedSchema } from '../../../../../main/src/plugin/configuration-registry';
+import NumberItem from './NumberItem.svelte';
+
+beforeAll(() => {
+  (window as any).getConfigurationValue = vi.fn();
+});
+
+test('Expect tooltip if NaN provided as value', async () => {
```

Comment:
hello, it looks it's missing test that clicks on increment/decrement (and if we can increment or decrement when we reach min/max value)

Reply:
done